.env_sample

```diff
@@ -1,6 +1,7 @@
 # App domain + Protocol setting for development
 export APP_DOMAIN="localhost:3000"
 export APP_PROTOCOL="http://"
+export APP_NAME="forem_local"
```

Comment:
what if we use the hostname or app domain instead of a colloquial name?

Reply:
It doesn't _really_ matter because we could probably change this if we made that change, but the way we're using `APP_NAME` in production as a unique identifier which sticks with the forem forever whereas the domain could hypothetically be a bit more fluid, or one app could have multiple domains (not something we support in any capacity now, but in some scenario like `smile.amazon.com` functionality or whatever, I just feel like systems-wise this is the proper thing.

FYI `APP_NAME` might be new to local, but had already been an environment variable that existed in Heroku land and we adopted as a variable in Forem Cloud universe, in case that info is not totally clear as well.

Reply:
thanks for the explanation, makes sense!